Control/public/common/detectorModal.js

```diff
@@ -32,20 +32,24 @@ const detectorsModal = (model) =>
         model.detectors.listRemote.match({
           NotAsked: () => null,
           Loading: () => h('.w-100.text-center', loading(2)),
-          Success: (data) => detectorsList(model, data),
+          Success: (data) => detectorsList(model, model.session.admin ?
```

Comment:
I think it would be cleaner if we move the detector logic on what detectors are sent in the `detectorList` and we keep the `match` statement simpler.